webapp/app.py

```diff
@@ -57,4 +142,20 @@ def process_invoices_status_data(task_id: str) -> Response:
 
         return jsonify({"status": task_status, "logs": logs})
 
+    @app.route("/logout")
+    @login_required
+    def logout() -> str:
+        """Log out user by clearing Flask session and cookies.
+
+        Note: This doesn't log the user out of ALB or Touchstone.
+        """
+        # clear Flask session
+        session.pop("oidc_access_token", None)
+        session.pop("user", None)
+
+        # clear cookies
```

Comment:
Nit picky: I think of this as invoking the `flask-login` logout behavior, whatever it is that it does™️

Reply:
Please follow up in https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60#discussion_r1723876511.